cabal.project

```diff
@@ -59,205 +68,227 @@ package cardano-wallet-launcher
 package cardano-wallet-core-integration
   optimization: False
 
-package cardano-crypto-praos
-  flags: -external-libsodium-vrf
-
--- bring in plutus-apps that we need; then we copy the rest of the cabal.project from them
+-- Direct dependency.
+-- Are you thinking of updating this tag to some other commit?
+-- Please ensure that the commit you are about to use is the latest one from
+-- the *develop* branch of this repo:
+--   * <https://github.com/input-output-hk/iohk-monitoring-framework/commits/develop>
+-- (not master!)
+--
+-- In particular we rely on the code from this PR:
+--  * <https://github.com/input-output-hk/iohk-monitoring-framework/pull/622>
+-- being merged.
```

Comment:
It is weird to have a comment stating that we rely on a develop version because of a specific PR, whereas this PR was merged before the Audit Team was even created.

Reply:
This was copied from `plutus-apps`, I'll wait for CI to turn green so it caches the build and I'll add a comment to our `cabal.project` stating what was copied and what is ours.